client/src/App.jsx

```diff
@@ -3,7 +3,11 @@ import './App.css';
 import TextForm from './components/TextForm';
 import { ConnectionManager } from './components/ConnectionManager';
 import { socket } from './socket'
-
+import Header from './components/Header';
+import Login from './components/Login';
+import Main from './components/Main';
+import Footer from './components/Footer';
+import { Routes, Route } from 'react-router-dom';
 
 function App() {
   const [passage, setPassage] = useState("");
```

Comment:
we can discuss which component this should live in

Reply:
YUH